src/Dev/Tasks/TagsToShortcodeHelper.php

```diff
@@ -253,7 +253,7 @@ private function getParsedFileIDFromSrc($src)
 
         // set fileID to the filepath relative to assets dir
         $pattern = sprintf('#^/?(%s/?)?#', ASSETS_DIR);
-        $fileID = preg_replace($pattern, '', $src);
+        $fileID = preg_replace($pattern ?? '', '', $src ?? '');
```

Comment:
```suggestion
        $fileID = preg_replace($pattern ?? '//', '', $src ?? '');
```

Reply:
Though in this case we can just trust that it's a valid regex based on how it's assigned above.